types/react-calendar-heatmap/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for react-calendar-heatmap v1.6.1
+// Project: https://github.com/patientslikeme/react-calendar-heatmap
+// Definitions by: Keisuke Kan <https://github.com/9renpoto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="react" />
```

Comment:
If you move everything to the top level, you should just import React here instead of using the `/// <reference` comment.